db/re/skill_db.conf

```diff
@@ -18943,6 +18943,36 @@ skill_db: (
 	Description: "Wide Charm"
 	MaxLevel: 5
 },
+{
+	Id: 708
```

Comment:
Shouldn't it also include `ShowSkillScale` ? as rA has `ShowScale`

Reply:
Done